README.md

```diff
@@ -33,9 +33,13 @@ These instructions will get you a copy of the project up and running on your loc
 
 * [Java 11](https://openjdk.java.net/projects/jdk/11/) or higher
 * [Apache Maven 3.5](https://maven.apache.org/download.cgi) or higher
-* [Lombok 1.18.16](https://projectlombok.org/download) or higher. Please check the [requirements](https://projectlombok.org/setup/overview) for your IDE 
 * data: [download](http://downloads.ohsome.org/) it directly, or work through a guide on [how to prepare a new OSHDB extract](https://github.com/GIScience/oshdb/blob/master/oshdb-tool/etl/README.md)
 
+### IDE Settings
+
+* [Code style](/CONTRIBUTING.md#code-style): For some popular IDEs and code linting tools you can find configuration files of the used code style in the OSHDB repository: [config/ide](https://github.com/GIScience/oshdb/tree/master/config/ide).
+* [Lombok](https://projectlombok.org/): Please check the [setup instructions](https://projectlombok.org/setup/overview) for your IDE. Note that you might have to mark the `src/main/lombok` directory as a _sources_ directory and _unmark_/_exclude_ `target/generated-sources/delombok`.
```

Comment:
Are all (common) IDEs marking the resources directories properly? At least IDEA does that, I think :)
One could add something like excluding `docs/_build` as well as this is not standard Java. And I don't know about the `ignite` folder, the ohsome API creates when using ignite. Probably it would be better to exclude them as well, as they don't contain valuable information that needs to be indexed.

Reply:
I'm also not sure if it's really necessary, but I guess what you propose would not hurt.

Can you make a suggestion for the mentioned changes?

Reply:
Even the condensed one is very extensive :D